Gemfile.lock

```diff
@@ -343,4 +343,4 @@ DEPENDENCIES
   xmlrpc (~> 0.3.3)
 
 BUNDLED WITH
-   2.5.18
+   2.7.2
```

Comment:
We need to upgrade to 2.5.19 to fix https://github.com/ruby/rubygems/issues/7983

See: https://github.com/beefproject/beef/actions/runs/19336406639/job/55312500339?pr=3445

May as well upgrade to the latest 🤷🏽‍♂️